CHANGES.md

```diff
@@ -15,6 +15,7 @@ There are likely to be descriptions etc outside of the list below, but new query
 | [#289](https://github.com/ethereum/beacon-APIs/pull/289) `/eth/v2/beacon/blocks/{block_id}` added `deneb` block to response                 |                                                  |                                                   |                                                    |                                                 |                                           |
 | [#289](https://github.com/ethereum/beacon-APIs/pull/289) `/eth/v1/validator/blinded_blocks/{slot}` added `deneb` blinded block              |                                                  |                                                   |                                                    |                                                 |                                           |
 | [#289](https://github.com/ethereum/beacon-APIs/pull/289) `/eth/v1/validator/blocks/{slot}` added `deneb` block                              |                                                  |                                                   |                                                    |                                                 |                                           |
+| [#286](https://github.com/ethereum/beacon-APIs/pull/286) NEW `/eth/v1/beacon/blobs/{block_id}`                                              |                                                  |                                                   |                                                    |                                                 |                                           |
```

Comment:
will want to update this if we're changing path...

Reply:
Updated in a6d1c360991676c36cc5691a17d8ac1c17813c33